mediorum/ddl/add_fields_to_uploads.sql

```diff
@@ -0,0 +1,8 @@
+begin;
+
+ALTER TABLE uploads
```

Comment:
wondering about backwards compatibility here. I think it should be fine, but maybe worth testing when merging to staging by releasing this change manually to only some nodes but not others, and then uploading a track with a preview. I'd be curious to see if there are any logs for `invalid crud data` or if the new fields are properly ignored on non-updated nodes when crudr does `reflect.New(reflect.SliceOf(elemType)).Interface()`

Reply:
I think it should be fine -- `elemType` is `c.typeMap[op.Table]` so the records from `records := reflect.New(reflect.SliceOf(elemType)).Interface()` line should remain the same on non-updated nodes. Then it unmarshals `op.Data` [here](https://github.com/AudiusProject/audius-protocol/blob/84cf00dc30fe07c25b7b91ef43739786821b3ce7/mediorum/crudr/crudr.go#L213) and I know that `json.Unmarshal` ignores extra fields and doesn't error.

but despite there being no errors, we should only expect this feature to fully work when they're all updated, i.e. don't send requests with `preview_start_seconds` until this change is on all nodes to avoid data inconsistency + missed transcoding tasks.

but just in case will deploy to only a subset of stage nodes first.